DotnetRuntimeMetadata.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-macos`
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected exactly '127.0.0.1', but got 10.79.9.91.
at <ScriptBlock>, /Users/runner/runners/2.166.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 258
258:             $result[0].TargetAddress | Should -BeExactly $realAddress
```